ansible/roles/ocp4-workload-migration/tasks/remove_workload.yml

```diff
@@ -1,12 +1,15 @@
 ---
-# Implement your Workload removal tasks here
-
-- name: "Removing controller and operator"
-  debug:
-    msg: "Not implemented"
+- name: "Removing workload"
+  include_tasks: "{{ item }}"
+  with_items:
+    - './pre_workload.yml'
+    - './workload.yml'
+    - './post_workload.yml'
+  vars:
+    migration_workload_destroy: yes
 
 # Leave this as the last task in the playbook.
 - name: remove_workload tasks complete
   debug:
     msg: "Remove Workload tasks completed successfully."
-  when: not silent|bool
+  when: not silent|bool
```

Comment:
Why remove the newline?
Generally missing newlines at the end of file are considered a type of whitespace error I believe.

Reply:
My guess is that my editor (vs code) removed this automatically, I'll check on adding it back.

Reply:
Believe my editor did this automatically, I'll see about adding whitespace back.

Reply:
Added back.